plugins/ToolStatus/toolstatus.h

```diff
@@ -115,7 +115,8 @@ typedef union _TOOLSTATUS_CONFIG
         ULONG ResolveGhostWindows : 1;
         ULONG ModernIcons : 1;
         ULONG AutoHideMenu : 1;
-        ULONG Reserved : 4;
+        ULONG SearchAutoFocus : 1;
+        ULONG Reserved : 3;
```

Comment:
You'll need to use the Spare bits rather than Reserved since they were used by 2.39 and would conflict.